src/api/index.ts

```diff
@@ -0,0 +1,15 @@
+// eslint-disable-next-line
+export type Json = Record<string, any>;
+export type Response = Promise<Json>;
+
+export const request = async (url: string): Response => {
+  try {
+    const response = await fetch(url);
+    if (!response.ok) throw new Error();
+    const json = await response.json();
+    if (typeof json === "object" && !Array.isArray(json)) return json;
+    else throw new Error();
+  } catch (error) {
+    throw new Error(`Couldn't complete request\n${url}`);
```

Comment:
Did you add "\n" in the middle on purpose? It seems a little weird.

Reply:
Yes, this was to put the request url on a new line